session.go

```diff
@@ -56,75 +57,49 @@ func (s Session) DefaultFrontpage() ([]*Submission, error) {
 	return submissions, nil
 }
 
-// SubredditSubmissions returns the submissions on the given subreddit.
-func (s Session) SubredditSubmissions(subreddit string) ([]*Submission, error) {
+func (s Session) SortedSubmissions(subreddit string, sort popularitySort, config map[string]string) ([]*Submission, error) {
```

Comment:
Can we renamed `config` to `params`? What kind of parameters are you passing here; can we do something more specific than a generic map[string]string?